demos/image-classification/index.js

```diff
@@ -181,6 +183,18 @@ const main = async () => {
     }
   } catch (err) {
     log(`[Error] ${err}`);
+
+    status.setAttribute("class", "red");
```

Comment:
Does "red" map to the same pinkish color used in the existing error messages?

![image](https://github.com/user-attachments/assets/ed836df6-8356-4ef6-ba04-16a00a31e582)

Reply:
It's mapped. Actually I used "red" as the class name for errors, but ended up using pink for the actual color schema. Sorry for the misunderstanding.

#webnnstatus.red {
    background-color: var(--pink)
}

Reply:
> It's mapped.

👍 Resolving.